docs/output.md

```diff
@@ -4,8 +4,6 @@ This document describes the output produced by the pipeline. Most of the plots a
 
 The directories listed below will be created in the results directory after the pipeline has finished. All paths are relative to the top-level results directory.
 
-<!-- TODO nf-core: Write this documentation describing your workflow's output -->
-
 ## Pipeline overview
 
 The pipeline is built using [Nextflow](https://www.nextflow.io/)
```

Comment:
is this file up to date? talks about FastQC and stuff.
Does the output need to be documented here?

Reply:
I didn't document the output yet its still the default. I would do this in a follow up PR as it might still change.